mflowgen/core/run.py

```diff
@@ -151,13 +159,28 @@ def launch( s, help_, design, update=False, subgraph=False, backend='make' ):
   # graph description.
   #
 
-  def launch_run( s, design, update, subgraph, backend ):
+  def launch_run( s, design, update, subgraph, backend, graph_kwargs ):
 
     # Find the construct script (and check for --update) and save the path
     # to the construct script for future use of --update
 
     construct_path = s.find_construct_path( design, update )
-    s.save_construct_path( construct_path )
+
+    # If update is true and no new graph_kwargs are provided,
+    # look for the previous graph_kwargs
+
+    if update and not graph_kwargs:
```

Comment:
Can you make a separate `find_graph_kwargs()` that allows you to mirror the find construct path above?

graph_kwargs = s.find_graph_kwargs( ... )